content/en/examples/controllers/job-pod-failure-policy-example.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: job-pod-failure-policy-example
+spec:
+  completions: 12
+  parallelism: 3
+  completionMode: Indexed
```

Comment:
It doesn't need to be Indexed. I would use a regular job just to simplify the example.

Reply:
I know, I was thinking to demonstrate with this example somehow that the different parallel pod instances do something else. But maybe indeed this is unnecessary complication for the example.